packages/library-base/src/commonMain/kotlin/io/realm/internal/RealmObjectHelper.kt

```diff
@@ -329,7 +329,10 @@ internal object RealmObjectHelper {
     }
 
     @Suppress("UNUSED_PARAMETER")
```

Comment:
Is this annotation still relevant?

Reply:
Yes. It is only invoked from compiler plugin generated code. I will add a comment.